cmd/agent/client/client.go

```diff
@@ -0,0 +1,108 @@
+package client
```

Comment:
[CHANGE REQUEST] Please find more suitable location for this package.
https://github.com/golang-standards/project-layout